### PR TITLE
Make english

### LINKED ISSUE #5: Make English version

### PR DESCRIPTION
**Pull request by andrewslotin**
Отличный гайд, которым я бы очень хотел поделиться со своими коллегами, которые боятся make как огня. Коллеги хорошие, только вот по-русски не говорят, поэтому я перевел написанное на английский и готов это сделать для будущих глав тоже.

Closes #5

### CODE REVIEW

**Comment by inem** [comment]
Ух, круто, спасибо!

File: modern-make-handbook-en.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+```
+
+This does not look right. If your text editor can recognize different file formats, it probably already figured out that it should use tabs. Otherwise, you'll need to tweak it a little.
+
+Now, when Bob went down this rabbit hole and learned all the tricks, he can move on.
+
+<div style="page-break-after: always;"></div>
+## 3. Running multiple commands at once
+
+What if we want to run the tests, and if they pass deploy our code? No problemos:
```
Comment (llelf) [inline modern-make-handbook-en.md]: No problems?

File: modern-make-handbook-en.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+	@make schema-reset
+```
+
+Yay!
+
+
+<div style="page-break-after: always;"></div>
+## 8. Ignoring errors
+
+Bob's team decided to run tests befo deploing on staging as well:
```
Comment (llelf) [inline modern-make-handbook-en.md]: before deploying

File: modern-make-handbook-en.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+<div style="page-break-after: always;"></div>
+## 1. Making your library of shortcuts
+
+– What does the cat say when it's hungry?
+
+– Meow!
+
+
+
+– What does the dog way when it's warning?
```
Comment (llelf) [inline modern-make-handbook-en.md]: feeling danger?

File: modern-make-handbook-en.md (hunk 1 above)
Comment (andrewslotin) [reply modern-make-handbook-en.md]: I decided to keep the Spanish phrase that is used in Russian version.

Comment (llelf) [reply modern-make-handbook-en.md]: Ah right.
But shouldn’t it be “No problemas”?

File: modern-make-handbook-en.md (hunk 3 above)
Comment (andrewslotin) [reply modern-make-handbook-en.md]: I used "it is warning" here for brevity. A literal translation would be "What does the dog say when it wants to warn about danger". The shorter version has pretty much the same meaning, while "What does the dog say when it feels danger" is more abstract and does imply a single answer (the dog can whine, bark, roar, etc.)

File: modern-make-handbook-en.md (hunk 1 above)
Comment (andrewslotin) [reply modern-make-handbook-en.md]: Right, either "no problemas", which is a correct Spanish spelling, or "no problemo" for an [American slang version](https://www.urbandictionary.com/define.php?term=No%20Problemo). I'd probably go for the latter 👍

**Comment by andrewslotin** [comment]
@llelf, @inem, is there anything else that needs to be changed before this PR is merged?

**Comment by inem** [comment]
Я в процессе редактирования перевода. Думаю сегодня-завтра зарелизить!

**Comment by inem** [comment]
Closing in favor of https://github.com/inem/make-handbook/pull/11

**Pull request closed without merging**